internal/mongo/event.go

```diff
@@ -0,0 +1,50 @@
+package mongo
+
+import (
+	"fmt"
+
+	"go.mongodb.org/mongo-driver/bson"
+	"go.mongodb.org/mongo-driver/bson/primitive"
+)
+
+const documentKeyField = "documentKey"
+const idField = "_id"
+
+// check if a change event if valid
+// A valid event has at least a document key value
+func isValid(event bson.M) error {
+	_, err := documentID(event)
+	return err
+}
+
+// marshall event to an array of bytes
+func marshall(event bson.M) ([]byte, error) {
```

Comment:
I think this function name should took only 1 `l` letter? (marshal / marshalling), it's like `travail` and `travailler` in french
Marshall is the sound products company 😄 (https://marshall.com)